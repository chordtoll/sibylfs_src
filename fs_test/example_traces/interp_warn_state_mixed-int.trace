@type script
open /foo [O_EXCL]
