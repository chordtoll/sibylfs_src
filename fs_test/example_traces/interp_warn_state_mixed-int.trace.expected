# processing file 'example_traces/interp_warn_state_mixed-int.trace' ...
        @type trace
     2: open "/foo" [O_RDONLY;O_EXCL]
# 
# Error:      2: open "/foo" [O_RDONLY;O_EXCL]
#    special states found:
#      - OS_special (Undefined, "open: O_EXCL, no O_CREAT, posix/open.md D:1")
#    continuing with normal result states
        Tau
        ENOENT
# Warning: trace caused problems
# processing file 'example_traces/interp_warn_state_mixed-int.trace' failed
